src/sock.c

```diff
@@ -503,8 +503,8 @@ static void mg_iotest(struct mg_mgr *mgr, int ms) {
 #elif MG_ENABLE_POLL
   nfds_t n = 0;
   for (struct mg_connection *c = mgr->conns; c != NULL; c = c->next) n++;
-  int pollCount = n == 0 ? 1 : 1; // Avoid zero-length VLA
-  int pollSize = pollCount * (int) sizeof(struct pollfd);
+  size_t pollCount = n == 0 ? 1 : 1; // Avoid zero-length VLA
```

Comment:
Probably it should be `size_t pollCount = n == 0 ? 1 : n; // Avoid zero-length VLA`